src/extra/dwave/chimera.jl

```diff
@@ -0,0 +1,170 @@
+        )
+    end
+
+    cell_row_size = cell_size ÷ 2
+
+    # These values are used to transform a variable index into a chimera 
+    # coordinate (x,y,o,i)
+    # x - row
+    # y - col
+    # o - cell_col - indicates the first or the second row of a chimera cell
+    # i - cell_col_id - indicates ids within a chimera cell
+    # Note that knowing the size of source chimera graph is essential to doing this mapping correctly
+
+    cell        = Dict{Int,Int}(i => (i ÷ cell_size) for i in 1:dimension)
+    row         = Dict{Int,Int}(i => (j ÷ degree) for (i, j) in cell)
+    col         = Dict{Int,Int}(i => (j % degree) for (i, j) in cell)
+    cell_col    = Dict{Int,Int}(i => (i % cell_size) ÷ cell_row_size for i in 1:dimension)
+    cell_col_id = Dict{Int,Int}(i => (i % cell_row_size) for i in 1:dimension)
+    
+    coordinates = Dict{Int,NTuple{4,Int}}(
+        i => (row[i], col[i], cell_col[i], cell_col_id[i])
+        for i in 1:dimension
+    )
+
+    # Get the maximum of both row id and column id (namely 'c[1]', 'c[2]')
+    effective_degree = 1 + maximum((_, c) -> max(c[1], c[2]), coordinates)
+
+    if effective_degree > degree
+        error(
+            """
+            The effective degree '$effective_degree' value is greater than the chimera degree '$(degree)', which is infeasible.
+            """,
+        )
+    end
+
+    return Chimera(
+        grid_size,
+        cell_size,
+        precision,
+        coordinates,
+        degree,
+        effective_degree,
+    )
+end
+
+function DWaveDevice(arch::Chimera, model::QUBOTools.AbstractModel{V}) where {V}
+    L = collect(QUBOTools.linear_terms(model))
+    Q = collect(QUBOTools.quadratic_terms(model))
+    α = QUBOTools.scale(model)
+    β = QUBOTools.offset(model)
+
+    γ = max( # maximum absolute value for a coefficient in 'model'
+        maximum((_, v) -> abs(v), L),
+        maximum((_, v) -> abs(v), Q),
+    )
+
+    γχ =  10 ^ arch.precision / γ
+
+    # Chimera Coefficients
+    Lχ = sizehint!(Dict{V,Int}(), length(L))
+    Qχ = sizehint!(Dict{Tuple{V,V},Int}(), length(Q))
+
+    for (i, v) in L
+        xi = QUBOTools.variable(model, i) 
+
+        Lχ[xi] = round(Int, v * αχ)
+    end
+
+    for ((i, j), v) in Q
+        xi = QUBOTools.variable(model, i)
+        xj = QUBOTools.variable(model, j)
+
+        Qχ[(xi, xj)] = round(Int, v * αχ)
+    end
+
+    αχ = 1
+    βχ = round(Int, β * γχ)
+
+    factor = α * γχ
+
+    return DWaveDevice(
+        arch,
+        QUBOTools.Model{V,Int,Int}(
+            Set{V}(QUBOTools.variables(model)),
+            Lχ,
+            Qχ;
+            scale    = αχ,
+            offset   = βχ,
+            sense    = QUBOTools.sense(model),
+            domain   = QUBOTools.domain(model),
+            metadata = copy(QUBOTools.metadata(model)),
+            start    = QUBOTools.start(model),
+        ),
+        factor,
+    )
+end
+
+function layout(arch::Chimera)
+    # TODO: Retrieve this information from dwave-networkx?
```

Comment:
They provide functions to generate the layouts. I think it might be faster to just reimplement them in the future (great 1st PR for someone artistic) to do

Reply:
The idea here is to retrieve the `networkx` graph from dwave and convert it into a `Graphs.jl` object. That should be enough for generating those visualizations you saw earlier today.